manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/workers/quartz/WorkJob.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.service.smartevents.manager.v2.workers.quartz;
+
+import javax.inject.Inject;
+
+import org.quartz.Job;
+import org.quartz.JobDataMap;
+import org.quartz.JobExecutionContext;
+
+import com.redhat.service.smartevents.infra.core.exceptions.definitions.platform.InternalPlatformException;
+import com.redhat.service.smartevents.manager.core.workers.Work;
+import com.redhat.service.smartevents.manager.core.workers.Worker;
+import com.redhat.service.smartevents.manager.v2.persistence.models.Bridge;
+import com.redhat.service.smartevents.manager.v2.workers.resources.BridgeWorker;
+
+import static com.redhat.service.smartevents.manager.core.workers.quartz.QuartzWorkConvertor.convertFromJobData;
+
+/**
+ * Single Job implementation that invokes the applicable {@link Worker} based on the {@link JobDataMap} .
+ */
+public class WorkJob implements Job {
+
+    private static final String BRIDGE_WORKER_CLASSNAME = Bridge.class.getName();
+
+    @Inject
+    BridgeWorker bridgeWorker;
```

Comment:
We did speak about injecting `Instance<Worker>` and delegate the [matching](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1396/files#diff-ae50e62edc4a19cb423e3324209671370933935f0d0fd25ad5df37b850c7c8fdR36) to the `Workers`...

... but it's OK if you wanted to keep it this way in `v2`.